hologram_dashboard/README.md

```diff
@@ -0,0 +1,20 @@
+## Preparation
+    1.  clone the repository with "git clone 
+        https://github.com/googleinterns/sarahchen-intern-2020.git
+    2.  navigate into the "hologram_dashboard" folder with 
+        "cd hologram_dashboard"
+
```

Comment:
3. Install bazel
    XXX

Reply:
bazel installation is dependent on the system the user is operating on should I just assume for ubuntu?